docker/nginx-default.conf

```diff
@@ -0,0 +1,76 @@
+server {
```

Comment:
Seeing the `cert_setup` scrips and "custom" nginx configuration: For try.zeek.org, Justin opted to use [caddy](https://caddyserver.com/) that might simplify this a bit:

https://github.com/zeek/try-zeek/blob/617912cbacc056ca38941b340715014c678f953d/docker-compose.yml#L47-L53

https://github.com/zeek/try-zeek/blob/master/containers/caddy/Caddyfile.tls

Did you opt against it? Or is the current deployment nginx based and the configuration below is known to work and switching server software too much of a hurdle?

Reply:
The existing deployment is a hairball of apache configuration, and switching to nginx was considerably easier than duplicating that in Docker. I didn't look at Caddy, since I know nginx well enough to just bang it out.